gluten-core/src/main/java/io/glutenproject/row/RowIterator.java

```diff
@@ -21,11 +21,18 @@
 
 import java.io.IOException;
 
-public class RowIterator {
-  private long nativeHandler = 0L;
+/**
+ * This is a built-in native implementation of {@link BaseRowIterator}. This is a part of the
+ * provided {@link io.glutenproject.vectorized.NativeExpressionEvaluator} implementation as
+ * JNI facility for wiring with native codes.
+ */
+// FIXME can we rename this by adding a "native" prefix to make it clear the iterator is
```

Comment:
This class will be removed later, I think it's not needed.

Reply:
👍